cookiecutter/README.md

```diff
@@ -0,0 +1,18 @@
+# Streamlit template for CookieCutter
+
+Template for creating a streamlit component
+
+## Usage
+
+Make sure you have [https://pypi.org/project/cookiecutter/](cookiecutter) installed. To install it, you can use [pipx](https://pypa.github.io/pipx/):
+```sh
+pipx install cookiecutter
+```
+
+Then run
+// TODO: Remove --checkout cookieuctter before merge
+```
+cookiecutter https://github.com/streamlit/component-template.git --directory=cookiecutter --checkout cookieuctter
```

Comment:
`--checkout cookieuctter`

should be `--checkout cookiecutter`

Reply:
Yes, but this parameter will be deleted before merging anyway.

Reply:
No harm in correcting the typo IMO :)